internal/manifests/var.go

```diff
@@ -80,7 +80,7 @@ func commonLabels(stackName string) map[string]string {
 func serviceAnnotations(serviceName string, enableSigningService bool) map[string]string {
 	annotations := map[string]string{}
 	if enableSigningService {
-		annotations[openShiftServingCertKey] = signingServiceSecretName(serviceName)
+		annotations[openshift.ServingCertKey] = signingServiceSecretName(serviceName)
```

Comment:
This is mainly just a thought. Should this method be added into the `internal/manifests/openshift/` directory because this is an OCP feature?